deluge/common.py

```diff
@@ -900,6 +900,16 @@ def free_space(path):
         return disk_data.f_bavail * block_size
 
 
+def verify_ip_or_interface(ip):
```

Comment:
I think we should standardise on the naming. So interface in libtorrent terms is an IP or Adapter therefore `is_interface` would suffice perhaps?